src/AspNet.Security.OAuth.Ebay/AspNet.Security.OAuth.Ebay.csproj

```
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net5.0</TargetFrameworks>
```

This is used to make upgrades (e.g. from .NET 5 to .NET 6) easier to maintain without having to touch every project file.

```suggestion
    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
```